plugin/plugin.go

```diff
@@ -52,7 +54,7 @@ func init() {
 
 			snapshotter, err := aufs.New(root)
 			if err != nil {
-				return nil, errors.Wrap(plugin.ErrSkipPlugin, err.Error())
+				return nil, fmt.Errorf("%s: %w", plugin.ErrSkipPlugin.Error(), err)
```

Comment:
Apologies, I was wrong you had it right the first time with fmt.Errorf("%s: %w", err.Error(), plugin.ErrSkipPlugin

Reply:
the reason is we want to be able to check with is against ErrSkipPlugin and with this error wrapping model you can't nest errors other than with text prefixes..

Reply:
Done